### ISSUE TITLE
j M Y date format removes number portion of date from article card

### ISSUE THREAD

**Issue by JillChongva** [andyhqtran/divi-100-article-card#4]
### Problem:

I use j M Y as the date format on my blog.  Doing so name the number part of the date on the article card disappear.

### Steps To Reproduce:
1. Using blog module, grid
2. Change date format to j M Y
3. Publish
4. Two letter month abbreviation will show by number date will not
5. Issue resolves when date is changed back to M j, Y

### Additional Information:

**Device**: MacBook Pro Retina

**Operating System**:  OS X El Capitan

**Browser**: Safari or FF

**Screen Resolution**: 2880 x 1800

**Comment by tomatpercipi** [comment]
exactly same issue here (Chrome on W10), 
The M j, Y solution results in some errors too for me, only deleting completely the date formatting in blog module solved the problem.

**Comment by andyhqtran** [comment]
The plugin is set to read a specific date at this time.

-

@tomatpercipi is your site in another language? That seems to be causing issues.

**Comment by tomatpercipi** [comment]
Yes as DIvi is multilanguage. Mine is French. certainly the cause, agree.

**Comment by PabloMoratinos** [comment]
Hello mi Divi is spanish, my date format j M, Y and I have the same issue: the j doesn't show. The plugin  is the last version.
![image](https://cloud.githubusercontent.com/assets/20186785/16415053/8a84afe0-3d3b-11e6-8190-4cb18d46c42a.png)

**Comment by erikgeurts** [comment]
In my case, the site's Date format is set to "F j, Y". The site's language is English.

The blog module's data format is set to "M j, Y". The way it shows up on the site's front end is like this:
![image](https://cloud.githubusercontent.com/assets/5102461/16415680/c0940a24-3d3e-11e6-8f65-8cfeea2eda6d.png)

As we can see, the year is missing (but there wouldn't be enough space for it anyway).

**Comment by Aminericher** [comment]
Hi, the date does not appear as a card in my case. I use Divi in an arabic blog (rtl)
![date divi](https://cloud.githubusercontent.com/assets/8338017/17457460/e81d8388-5bf0-11e6-9850-f424b93ca531.png)
